### DISCUSSION TITLE
Metadata Saver: Does JF read and write and update `.nfo` files automatically?

### DISCUSSION THREAD

**Discussion by lamyergeier** [jellyfin/jellyfin#13302] [Q&A]
If I enable Metadata Saver, will Jelly fin read and write and update `.nfo` files automatically or may be periodically?

For example, if I make changes to the generated `.nfo` from a text editor, will those changes get relected in JF?

And if changes are made to metadata of already existing video from Jelly Fin Web Client, for which `nfo` file already exist, will the nfo file get updated?

Also can I save the nfo file as `Video_Filename.nfo`  instead of `movie.nfo`?

**Comment by TheMelmacian** [comment]
Yes, metadate changes via Jellyfin Web are directly written to the nfo, if enabled. Changes to the nfo file are not immediately detected by jellyfin. You can force the update by "scanning for new and updated files" on the affected item, or simply wait until jellyfin detects the changes.

If the nfo file is created by jellyfin, it will always be named movie.nfo. If you have multi version movies, the first entry is the movie.nfo (the main version) and all other versions will be named after the corresponding movie file.
If you create the nfo files yourself, you can use `Video_Filename.nfo`. But it's possible that jellyfin creates an additional `movie.nfo`.
